### PR TITLE
DT 2115: updated dependencies and updated helm/circleci/docker…

### PR DESCRIPTION
**Pull request by SimonMitchellMOJ**
…in line with our other projects

### CODE REVIEW

**Review by mikehalmamoj — DISMISSED** [review]

**Review by petergphillips — APPROVED** [review]
LGTM

File: helm_deploy/values-prod.yaml [hunk 1]
```diff
@@ -34,3 +34,8 @@ whitelist:
   cloudplatform-live1-1: "35.178.209.113/32"
   cloudplatform-live1-2: "3.8.51.207/32"
   cloudplatform-live1-3: "35.177.252.54/32"
+
+# determine which slack channel alerts are sent to, via the correct Alert Manager receiver
+generic-prometheus-alerts:
+  alertSeverity: digital-prison-service
```
[the remark this answers is not in the document]
Comment (SimonMitchellMOJ) [reply helm_deploy/values-prod.yaml:40]: I will have a look!

[the remark this answers is not in the document]
Comment (SimonMitchellMOJ) [reply helm_deploy/values-prod.yaml:40]: Looks like it is only in the values-prod.yaml

**Review by petergphillips — APPROVED** [review]
LGTM

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**